vpype/io.py

```diff
@@ -452,12 +459,27 @@ def write_svg(
             if len(line) <= 1:
                 continue
 
-            if len(line) == 2:
-                path = dwg.line((line[0].real, line[0].imag), (line[1].real, line[1].imag))
-            elif line[0] == line[-1]:
-                path = dwg.polygon((c.real, c.imag) for c in line[:-1])
+            if no_basic_shapes:
+                if len(line) == 2:
+                    path = dwg.path(d='M'+ str(line[0].real) + ',' + str(line[0].imag) + ' ' + str(line[1].real) + ',' + str(line[1].imag))
```

Comment:
I think this method of using `dwg.path` is not optimal. In particular `str()` doesn't handle precision the way SVG should (and the way functions like `dwg.line()` do). I'm unsure if `dwg.path` offers a better API for this, but in anycase we must avoid outputting too many decimals to keep file size in check.

Reply:
i did not find anything about decimals in https://svgwrite.readthedocs.io/en/latest/classes/path.html 

i had a look into the recent situation. default elements like line or polygon allow to have a lot of decimals too. I think it takes the decimalslike given in input:
![grafik](https://user-images.githubusercontent.com/5631071/114025044-fd67d200-9874-11eb-860f-ec4c3e8ded53.png)

but it looks like vpype cuts off after third decimal. So we could use '{:1.3f}'.format(value) to render 3 decimals for floating point or so.